src/core/blaze.py

```
@@ -1,16 +1,17 @@
-from src.core.block import BlazeBlock
-from src.core.seq import BlazeSequence
-from src.core.namegen import generate_scheduler_name
-from src.core._types import SubmitSequenceData, JobFile, BlazeLock, SequenceStatus, JobState
-from typing import List, Optional
-import os, multiprocessing, json, time, hashlib, signal, sys
+from .block import BlazeBlock
+from .seq import BlazeSequence
+from .namegen import generate_scheduler_name
+from ._types import SubmitSequenceData, JobFile, JobExecutuionData, BlazeLock, SequenceStatus, JobState
+from typing import List, Dict, Any, Optional
+import os, multiprocessing, json, time, logging, hashlib, signal, sys
```

Ruff (0.12.2)

Multiple imports on one line

Split imports

(E401)